Documentation/Extend reserve/UpdateTemplates/Index.rst

```diff
@@ -0,0 +1,47 @@
+.. include:: ../Includes.txt
+
+.. _update-templates:
+
+================
+Update templates
+================
+
+This extension uses fluid templates so you are able to override them like in most other TYPO3 extensions.
+
+Override template using TypoScript
+----------------------------------
+
+Do you have an site package? Put the templates into your site package extension. Otherwise you can put them into fileadmin if you really want.
```

Comment:
"an" --> "a"

Reply:
Mist :sweat_smile: da stand vorher "Do you have an extension" in meinem Entwurf